modules/wildfly/pom.xml

```diff
@@ -61,11 +61,6 @@
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
```

Comment:
+1 for not adding it to the dist.

But in case of the app server ZIPs, omitting it would leave behind the user
with a "half" ORM version, potentially falling onto their feet in case
they'd like to use ISPN as L2 cache. Not sure whether anyone is doing it,
but I'd leave this as is.